### ISSUE TITLE
wrong out features on cls_logits

### ISSUE THREAD

**Issue by fatalfeel** [AlphaJia/pytorch-faster-rcnn#11]
self.conv = nn.Conv2d(in_channels, in_channels, kernel_size=3, stride=1, padding=1)

'''background/foreground score'''
self.cls_logits = nn.Conv2d(in_channels, num_anchors, kernel_size=1, stride=1)  #wrong

'''bbox regression parameters'''
self.bbox_pred = nn.Conv2d(in_channels, num_anchors * 4, kernel_size=1, stride=1)

###################
self.cls_logits = nn.Conv2d(in_channels, num_anchors * 2, kernel_size=1, stride=1)  #correct
